test/with_qiskit/test_circuit_assert_equal.py

```diff
@@ -14,12 +14,12 @@ class TestCircuitAssertEqual(unittest.TestCase):
     How to execute this test:
     $ pwd
     {Your directory where you git-cloned quantestpy}/quantestpy
-    $ python -m unittest test.test_circuit_assert_equal
-    ...
+    $ python -m unittest test.with_qiskit.test_circuit_assert_equal
+    .s..........
```

Comment:
Why did you change line18?
What do you mean the new one?

Reply:
One dot `.` means one test, therefore # of dots increase as # of tests increases.